cmd/nox/snapshot.go

```diff
@@ -105,10 +108,17 @@ func init() {
 	snapshotDelete.Flags().StringVarP(&f, "frequency", "f", "", "Frequency of the snapshot")
 	snapshotDelete.Flags().StringVarP(&r, "repo", "r", "", "Repository for the snapshot")
 
+	snapshotStart.Flags().BoolVarP(&w, "wait", "w", false, "Wait for completion of snapshot")
+	snapshotStart.Flags().BoolVarP(&n, "notify", "n", false, "Send notification to slack")
+	snapshotStart.Flags().StringVarP(&s, "slack", "s", "", "Slack webhook to send snapshot failures to")
```

Comment:
this option is just the slack webhook url correct? do you need to pass in any like credentials or tokens or is it a public endpoint? 

Also, can this option be set in the `.nox` config file? or does it need to be passed in as an option to each call to the command? I can see it being useful to be able to configure one webhook endpoint for the everything. I think we can get that easily by binding it to a viper flag
```go
viper.BindPFlag("slack", snapshotStart.Flags().Lookup("slack"))
```

Reply:
I put it in /etc/elasticsearch/.nox.yaml. Should I make the slack argument a root option and have output from each of the commands go to slack if the argument is provided?
```viper.BindPFlag("slack", rootCmd.PersistentFlags().Lookup("slack"))
```
in nox/cmd/nox/root.go?

Reply:
huh I thought you had to manually bind the flag to `viper` for it to be usable from the config file. I must have misread the documentation. If it works, im good with how it is. 

and that is an interesting idea. I think I _may_ want to add some kind of alerting option down the line _maybe_ but for now lets leave it at just snapshots.